### PR TITLE
expose underlying HttpClient methods in HttpMethodsClient

### LINKED ISSUE #7: HTTP Methods Client implement HttpClient
Shouldn't the conventional HTTP method client implement the HttpClient interface and behave as a decorator of the underlying client?

### PR DESCRIPTION
**Pull request by dbu**
fix #7

### CODE REVIEW

File: src/HttpMethodsClient.php [hunk 1]
```diff
@@ -184,7 +207,7 @@ public function options($uri, array $headers = [], $body = null)
      */
     protected function send($method, $uri, array $headers = [], $body = null)
     {
-        return $this->httpClient->sendRequest($this->messageFactory->createRequest(
+        return $this->sendRequest($this->messageFactory->createRequest(
```
Comment (dbu) [inline src/HttpMethodsClient.php:190]: i guess we should do this, in case somebody extends the methods-client to add something on sendRequest

**Comment by sagikazarmark** [comment]
Why is the `send` method protected? I think it should be public.

File: src/HttpMethodsClient.php (hunk 1 above)
Comment (sagikazarmark) [inline src/HttpMethodsClient.php:190]: :+1:

**Comment by sagikazarmark** [comment]
:+1:

**Comment by dbu** [comment]
i did not change that. hm, but good question. the use case would be custom HTTP methods? probably makes sense, will change that.

i ran the tests locally. can we enable travis for this repo? the .travis.yml file is present already.

**Comment by sagikazarmark** [comment]
I know you didn't change that, just spotted it now.

Custom HTTP methods, GET requests with body.

Travis enabled.

**Comment by dbu** [comment]
made send public. now we also have a way to GET with a body without using the message factory. exposing the underlying HttpClient still makes sense.

**Comment by sagikazarmark** [comment]
:+1:

**Comment by dbu** [comment]
ups, i will try to fix the build.

**Pull request merged**